plocate: add page (#8541)
tldr-pages/tldr: pages/common/plocate.md

# plocate

> Find filenames quickly.

> > Make sure to use `sudo updatedb` to include new files.

Reviewer: ```suggestion
> Make sure to run `sudo updatedb` to include new files.
```

# plocate

> Find filenames quickly.
> More information: <https://plocate.sesse.net>.

> - Look for pattern in the database. Note: the database is recomputed periodically (usually weekly or daily):

Reviewer 2: I am not sure about this one maybe
```suggestion
- Look for patterns in the periodically recomputed database:
```
Or
```suggestion
- Look for patterns in the database (recomputed periodically):
```

`plocate {{pattern}}`

- Look for a file by its exact filename (a pattern containing no globbing characters is interpreted as `*pattern*`):

`locate */{{filename}}`

- Recompute the database (to find recently added files):

> `sudo updatedb`

Reviewer: We shouldn't be documenting other commands in the same page.

Reviewer 2: Agreed, I think we can remove this command from page if necessary.

Author: I think updating the database is important because, without it the **plocate** command won't work. Is it possible to add a link to another page in a page?

Reviewer: See `tldr fdisk` 

Within the description you can suggest to see other pages.

Author: It would take a lot of time to figure out the problem to new users if, they didn't notice that line. Is there any other way?

Reviewer: We could maybe add something to the description here such as, `make sure to use the updatedb to include new files`

Author: Looks good!

# plocate

> Find filenames quickly.

> > Make sure to use the `sudo updatedb` to include new files.

Reviewer: ```suggestion
> Make sure to run `sudo updatedb` to include new files.
```

Author: It's not getting committed as I've already removed 'the' from that line.

Author: You might have to suggest the changes to the latest version of the file
